spmd/tensor/api.py

```diff
@@ -304,6 +304,22 @@ def to_local(self) -> torch.Tensor:
         """
         return ToTorchTensor.apply(self)  # pyre-ignore[16]: autograd func
 
+    def to_logical_tensor(self) -> torch.Tensor:
```

Comment:
Can we call it "global tensor" instead?

Reply:
I was thinking about this more, and I choose to delete that API, and instead add a option to `to_local`, i think that might more sense to end user.